README.md

```diff
@@ -512,6 +516,208 @@ get '/', http_codes: [
 end
 ```
 
+## I18n
+
+Grape-swagger supports I18n for most messages of the JSON documentation, including:
+
+* API [info](#info)
+* Namespaces/resources description
+* Endpoints description and detail/notes
+* Params (including headers) description
+* Models/entities description
+
+By default, grape-swagger will lookup translations inside `:api` scope. This can be configured by
+set [`i18n_scope`](#i18n_scope) to a custom scope (or an array of scopes) when calling
+`add_swagger_documentation`.
+
+### Trigger Translation
+
+Take endpoint's description and detail for example, the none-i18n way to declare it with Grape can
+be done by:
+
+```ruby
+desc 'Get all kittens!' do
+  detail 'this will expose all the kittens'
+end
+get :kittens do
+end
+```
+
+To generate I18n documentation, you can set the description and detail to `nil` or blank string in
+your source code, but write the actual messages in locale file.
+
+Then you source code looks like this:
+
+```ruby
+# Either `nil` or blank string will trigger I18n translation.
+# And of course you can delete the desc block if there is nothing else except
+# desc and detail, because their default values are `nil`.
+desc nil do
+  detail ''  # Actually you don't need explicitly write this line.
+  # Some other things.
+end
+get :kittens do
+end
+```
+
+And in your locale file:
+
+```yaml
+api:
+  kittens:
+    get:
+      desc: Get all kittens!
+      detail: this will expose all the kittens
+```
+
+If a translation cannot be found, it will usually fallback to a blank string.
+
+### Customization
+
+It's very easy to customize the translation process, by set a non-trivial value to the message to be
+translated. As said above, `nil` or blank string message will trigger I18n translation, but it can
+be:
```

Comment:
You don't need to convince people that it's easy, just say what it is.

So, "To customize ... "